doc/policies/authentication.rst

```diff
@@ -390,6 +391,20 @@ type: bool
 If a user authenticates successfully all failcounter of all of his tokens
 will be reset. This can be important, if using empty PINs or *otppin=None*.
 
+always_increase_failcounter
```

Comment:
I would make clear, that this is in case of challenge response!

So the new policy should be named s.th. like `increase_failcounter_on_challenge_response`. (Or do I get s.th. wrong?)

Reply:
fixed in 4fe9f7f46e86b88dcd365672f86288d621345875.